index.html

```diff
@@ -104,6 +105,45 @@ <h5>Streaming vs. Saving</h5>
         <p class="link"></p>
       </div>
       <!-- End Streaming vs. Saving -->
+      
+      <!-- Troubleshooting Common Issues -->
+      <label class="recipe" for="troubleshooting">Trouble Shooting Common Issues</label>
+       <input type="checkbox" id="troubleshooting">
+      <div class="hiding"> 
+          <h5>Troubleshooting Common Issues</h5>
+          <p>There are many common issues you may encounter when using FFmpeg. This section is here to provide some possible explanations and solutions.For more general solutions to issues regarding download, installation, and other possible questions or issues, please refer to the <a href="https://ffmpeg.org/faq.html">FFmpeg FAQ.</a>
+<p>Before delving into some specific error messages and what may cause them, there are a few general tips you can follow to help make sure your files are doing what you want them to.
+First, always check that the output file actually opens, plays, looks, and/or sounds the way you intended it to. It is possible that FFmpeg will create an output file that, on the surface, looks correct, but upon further inspection, there may be inconsistencies with the codec, extension, or other file information, depending on what you intended to do with the file. 
+Second, always be certain that the file extension you are using for your output files is the intended extension. Within FFmpeg, certain extensions default to certain codecs, such as <code>.mp4</code> encoding to H.264. You always want to make sure that your extension coincides with your intended codec. A list of extensions and codec defaults can be found <a href="https://amiaopensource.github.io/ffmprovisr/#codec-defaults">here.</a></p>
+<h5>General Error Messages</h5>
+<h5>"Error: No such file or directory"</h5> 
+          <p><img src="img/directory_error_example.png" alt="no such file or directory error message"img></p> 
+          <p>There are a few reasons why this error may be appearing. First, check to make sure that you are in the correct directory. It can be easy to lose track of where your files are actually stored, especially on a personal computer. If you are working on a personal computer, the easiest way to do this is to check where your files are. When working with MacOS, for example, files are often stored in the downloads or documents folders. To change your directory from home to this specific directory, use the command <code>cd</code> followed by your desired folder. For downloads, it would look like this:  <code>cd downloads</code></p> 
```

Comment:
Just to remove uncertainty, I would phrase the last sentence as "For a folder named downloads, it would look like this"